lib/rides/pages/ride_detail_page.dart

```diff
@@ -151,13 +152,23 @@ class _RideDetailPageState extends State<RideDetailPage> {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageRideDetailTitle),
-        actions: <Widget>[
-          IconButton(
-            onPressed: () {},
-            icon: const Icon(Icons.chat),
-            tooltip: S.of(context).openChat,
-          )
-        ],
+        actions: _ride!.status == RideStatus.approved
```

Comment:
> Please always show the chat button. I think you should be able to explore the page even if there is no data yet, so that you can form a mental image of what the page will look like when it's full. I think it would benefit usability if you could click the button, and the "No chats yet" text on the chat page could even explain the relationship between rides and chats.

Same here

Reply:
i now changed it so that you see the chat when the ride was approved or is approved so that you can look at the chat after a ride/drive again in case you need that.